### PR TITLE
updated qlcolorcode (2.0.4)

### PR DESCRIPTION
**Pull request by tsparber**
This seems to be the most active fork. Original project seems to be abandoned. 

Tick #17075

### CODE REVIEW

File: Casks/qlcolorcode.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 cask 'qlcolorcode' do
-  version '2.0.2'
-  sha256 '317eda251ea5af8412401562395d2fbedb2dd915a7d927479cf09ac7251c4074'
+  version '2.0.4'
+  sha256 '6ae0b42c761181e493c94dcfabf22844859820055e77c780d92e56fc476fd9cb'
 
-  url "https://qlcolorcode.googlecode.com/files/QLColorCode-#{version}.tgz"
+  url 'https://github.com/anthonygelibert/QLColorCode/blob/master/bin/QLColorCode.qlgenerator.zip?raw=true'
```
Comment (jawshooah) [inline Casks/qlcolorcode.rb]: Downloading directly from `master` seems like a really bad idea.

Comment (tsparber) [inline Casks/qlcolorcode.rb]: I think so too. But this needs to be fixed upstream (I found no other download).

Comment (jawshooah) [inline Casks/qlcolorcode.rb]: I see. Then at least use an actual commit sha (`7cfb048` is the latest) instead of `master`, which can change at any time.

Comment (tsparber) [inline Casks/qlcolorcode.rb]: Good point. Amended

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 939c6d0d75e024d00a8fb4283e3c806e43fa4019 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**